cpp/src/arrow/ipc/read_write_test.cc

```diff
@@ -1334,7 +1334,7 @@ struct StreamDecoderWriterHelper : public StreamWriterHelper {
   Status ReadBatches(const IpcReadOptions& options, RecordBatchVector* out_batches,
                      ReadStats* out_stats = nullptr,
                      MetadataVector* out_metadata_list = nullptr) override {
-    auto listener = std::make_shared<CollectListener>();
+    auto listener = std::make_shared<CollectListener>(true);
```

Comment:
Please mention the parameter name
```suggestion
    auto listener = std::make_shared<CollectListener>(/*xxx=*/ true);
```

Reply:
Also, why do we set it to true here? Can you add a comment?